### PR TITLE
Remove missing license warning when pinning with opam

### PR DESCRIPTION
**Pull request by filipeom**
To remove the: 

```
  warning 68: Missing field 'license'
```

When using `opam pin` or `opam lint`. 

I know this PR might seem nitpicky, but that's not the intention. I'm just trying to keep opam happy

EDIT: Btw, while I'm here, are there any plans for an opam release in the future?

### CODE REVIEW

**Comment by filipeom** [comment]
[088428e](https://github.com/sneeuwballen/benchpress/pull/77/commits/088428ee87279c429796cf27d757f8ea4079eed6) should fix ci

File: src/server/benchpress_server.ml [hunk 1]
```diff
@@ -1746,9 +1746,9 @@ let handle_file self : unit =
       Log.debug (fun k -> k "get-file: `%s`" file);
       let bytes =
         if file = "prelude" then
-          H.Byte_stream.of_string Builtin_config.config (* magic file! *)
+          H.IO.Input.of_string Builtin_config.config (* magic file! *)
```
Comment (c-cube) [inline src/server/benchpress_server.ml:1749]: I think this needs an additional lower bound on  tiny_httpd ! (I think 0.14 would do it).

Comment (c-cube) [reply src/server/benchpress_server.ml:1749]: ha!, didn't see the last change

Comment (filipeom) [reply src/server/benchpress_server.ml:1749]: I think these changes are specific to the `tiny_httpd` version 0.17.0. It seems there's no longer `Tiny_httpd_stream` so I used `Tiny_httpd.IO.Input` instead.

**Comment by c-cube** [comment]
already merged in #78

**Pull request closed without merging**